StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButtonContent.swift

```diff
@@ -5,20 +5,43 @@
 //  Created by Nick Porter on 2/12/25.
 //
 
+@_spi(STP) import StripeUICore
 import UIKit
 
 /// Defines a view that is contained with a `RowButton`
-protocol RowButtonContent: UIView {
+protocol RowButtonContent: UIView, EventHandler {
```

Comment:
I'm growing more convinced that subclassing is the right strategy here. I feel like these RowButtonX classes *are* RowButtons. They'll always all share a common set of properties. They all have the same behaviors except for a handful which they override.

Reply:
Yeah, I agree that subclassing could work too, but I’m not seeing compelling benefits over our current composition approach, particularly given that we’d have to restart a good chunk of the refactor. Both approaches could scale similarly, though subclassing could turn RowButton into a ‘god class’ if more variations appear in the future, but composition isn’t immune to that risk either, but it does fit better with our existing pattern of avoiding UIKit subclasses when possible. Unless there’s a strong argument for subclassing, I’d prefer to stick with composition as we discussed. Definitely willing to hear the case for subclassing if you think there's some benefits!

Reply:
Benefits are that subclasses can inherit an required initializer so that you can't forget to handle a new subview in one particular RowButtonContent, subclasses can inherit and override and call super on methods like `setKeyContent`, they inherit all the shared properties (so we don't have to repeat the 8 properties 3-4 times).

It'd probably make most sense to try subclassing after you finish the third style.